subsys/bluetooth/services/ancs_attr_parser.c

```diff
@@ -88,6 +89,11 @@ static enum bt_ancs_parse_state app_id_parse(struct bt_ancs_client *ancs_c,
 					     const uint8_t *data_src,
 					     uint32_t *index)
 {
+	if (ancs_c->parse_info.current_app_id_index >= sizeof(ancs_c->attr_response.app_id)) {
+		LOG_DBG("App ID cannot be stored in response buffer.");
```

Comment:
Maybe warning log level would be better. What do you think ?

Reply:
Agree. Will do.